influxdb3_write/src/lib.rs

```diff
@@ -277,9 +282,13 @@ pub struct DatabaseTables {
     pub tables: hashbrown::HashMap<Arc<str>, Vec<ParquetFile>>,
 }
 
+/// The next file id to be used when persisting `ParquetFile`s
+pub static NEXT_FILE_ID: AtomicU64 = AtomicU64::new(0);
+
 /// The summary data for a persisted parquet file in a snapshot.
 #[derive(Debug, Serialize, Deserialize, Eq, PartialEq, Clone)]
 pub struct ParquetFile {
+    pub id: u64,
```

Comment:
Should this be a new type like `struct ParquetFileId(u64)`?

Reply:
I think to make it consistent it probably should be.

Reply:
I derived it and created some convenience methods to make it easier https://github.com/influxdata/influxdb/pull/25258/commits/ae0de02286305b81064eb65033e2e10844dc4acf